README.md

```diff
@@ -1,3 +1,20 @@
 # AlgebraicStockFlow
 [![Tests](https://github.com/AlgebraicJulia/StockFlow.jl/actions/workflows/tests.yml/badge.svg)](https://github.com/AlgebraicJulia/StockFlow.jl/actions/workflows/tests.yml)
 <!-- TODO: Set up on codecov.io for repo [![codecov](https://codecov.io/github/AlgebraicJulia/StockFlow.jl/branch/maaster/graph/badge.svg)](https://app.codecov.io/github/AlgebraicJulia/StockFlow.jl) -->
+
+StockFlow.jl is a Julia library for creating stock and flow diagrams using category theory.  Stockflows are used to represent systems where its population can move between different states, such as tracking how many people have been infected or recovered from a disease.  By providing initial values, and values for the parameters, we can solve the differential equation which the stockflow represents, which gives us a graph for how the population varies over time.
```

Comment:
For 'category theory', might it be helpful to link to the research and mention early on in the README some of the benefits of this model, like composition and stratification? On the WIP I wrote up, I had

> StockFlow.jl is a Julia library for the creation and execution of `[stock and flow modeling diagrams](https://en.wikipedia.org/wiki/System_dynamics#Stock_and_flow_diagrams)`, designed with model composition and stratification in mind through a `[categorical approach](https://arxiv.org/abs/2211.01290)`.

I think we want people that visit the library to know right away what it's about and why it's neat.

On the note of linking https://arxiv.org/abs/2211.01290 -- @Xiaoyan-Li thoughts on linking publications in the README for additional information?

Reply:
@Saityi yes, I agree with your suggestions. https://arxiv.org/abs/2205.08373 and https://arxiv.org/abs/2211.01290  are two papers we have worked on related to the current implement. I would definitely suggest to include to those two papers.